client/src/site-search/index.tsx

```diff
@@ -52,9 +52,9 @@ export function SiteSearch() {
       <PageContentContainer>
         {query ? (
           <h1>
-            Results: {query}{" "}
+            Search results for, <span className="query-string">{query}</span>{" "}
             {page && page !== "1" && (
-              <small className="current-page">Page {page}</small>
+              <span className="current-page">. Page {page}</span>
```

Comment:
I still think the page number thing should be more separated from the headline. It doesn't make sense to try to make the headline an English sounding sentence (not using a `:` and using a full-stop) but then write just "Page 3".

Would a screen reader sound terrible if it was:
```
Search results for: video (page 3)
``` 
?

Reply:
By the way, so we don't forget, the `current-page` class name is not used.